scripts/Dockerfile.test

```diff
@@ -46,7 +46,7 @@ RUN source /build.env \
     && mkdir -p ${GOROOT} \
     && curl https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-${GOARCH}.tar.gz  \
        | tar xzf - -C ${GOROOT} --strip-components=1 \
-    && curl -sf "https://install.goreleaser.com/github.com/golangci/golangci-lint.sh" \
+    && curl -sf "https://raw.githubusercontent.com/golangci/golangci-lint/${GOLANGCI_VERSION}/install.sh" \
```

Comment:
it's working locally. not sure what's wrong, running containerized-test it might take some time.

```text
[🎩︎]mrajanna@fedora ceph-csi $]cat test.sh 
#!/bin/bash
set -e
GOLANGCI_VERSION=v1.39.0

curl -sf "https://raw.githubusercontent.com/golangci/golangci-lint/${GOLANGCI_VERSION}/install.sh" | bash -s -- -b ${GOPATH}/bin "${GOLANGCI_VERSION}" 
[🎩︎]mrajanna@fedora ceph-csi $]./test.sh 
golangci/golangci-lint info checking GitHub for tag 'v1.39.0'
golangci/golangci-lint info found version: 1.39.0 for v1.39.0/linux/amd64
golangci/golangci-lint info installed /home/mrajanna/workspace/bin/golangci-lint
```

Reply:
This url is working for me as well.

Reply:
okay looks like the problem is with helm now

```
url -L hcurl -L https://git.io/get_helm.sh | bash -s -- --version v3.1.2
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   506  100   506    0     0     16      0  0:00:31  0:00:30  0:00:01   120
bash: line 1: syntax error near unexpected token `newline'
bash: line 1: `<!DOCTYPE html>'
```